### PR TITLE
Stats Benchmarks

### PR DESCRIPTION
**Pull request by dinooliva**
Adds microbenchmarks for OpenCensus Stats.

### CODE REVIEW

File: benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/SpanOperationsBenchmark.java [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    for (int i = 0; i < data.size; i++) {
+      span.addLink(data.links[i]);
+    }
+    return span;
+  }
+
+  private static AttributeValue[] createAttributeValues(int size) {
+    AttributeValue[] attributeValues = new AttributeValue[size];
+    for (int i = 0; i < size; i++) {
+      attributeValues[i] = AttributeValue.stringAttributeValue(ATTRIBUTE_VALUE + "-i");
```
Comment (rghetia) [inline benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/SpanOperationsBenchmark.java]: same as before. should it be ATTRIBUTE_VALUE + "-" + i ?

File: benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/BasicDataBenchmark.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  public Annotation createAnnotation(Data data) {
+    return Annotation.fromDescriptionAndAttributes(ANNOTATION_DESCRIPTION, data.attributeMap);
+  }
+
+  private static AttributeValue[] getAttributeValues(int size, String attributeType) {
+    AttributeValue[] attributeValues = new AttributeValue[size];
+    switch (attributeType) {
+      case "string":
+        for (int i = 0; i < size; i++) {
+          attributeValues[i] = AttributeValue.stringAttributeValue(ATTRIBUTE_VALUE_STRING + "-i");
```
Comment (rghetia) [inline benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/BasicDataBenchmark.java]: should this be ATTRIBUTE_VALUE_STRING + "-" + i ?

**Comment by rghetia** [comment]
Looks like rebase created a merge commit.

File: benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/SpanOperationsBenchmark.java (hunk 1 above)
Comment (dinooliva) [reply benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/SpanOperationsBenchmark.java]: Good catch - that's actually in the trace benchmarks, which seem to have appeared here with the rebase merge commit issue. I'll put together a new pr to fix the issue.

File: benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/BasicDataBenchmark.java (hunk 2 above)
Comment (dinooliva) [reply benchmarks/src/jmh/java/io/opencensus/benchmarks/trace/BasicDataBenchmark.java]: Similarly, this is part of the trace benchmarks, I'll put together a new pr to fix the issue.

**Comment by dinooliva** [comment]
Ta - thanks!

**Pull request merged**